src/internal.c

```diff
@@ -5319,6 +5321,8 @@ int SendBuffered(WOLFSSL* ssl)
                                 return WANT_WRITE;
                             }
                         }
+					#elif defined( FREERTOS_TCP )
+						return WANT_WRITE;	/* want write, call again   */
```

Comment:
Right, I will remove these lines.
WOLFSSL_CBIO_ERR_WANT_READ/WRITE will be returned